src/routes/Examples/components/Sidebar/Sidebar.jsx

```diff
@@ -22,7 +23,7 @@ const Sidebar = () => {
     };
 
     return (
-        <nav>
```

Comment:
Don't replace `<nav/>`, it has a semantic value for search engines. Either wrap it or have it wrap `<Scrollable/>`